launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -95,6 +95,11 @@ void CModListView::setupModsView()
 
 	ui->allModsView->setUniformRowHeights(true);
 
+	ui->allModsView->setContextMenuPolicy(Qt::CustomContextMenu);
+	
+	connect(ui->allModsView, SIGNAL(customContextMenuRequested(const QPoint &)),
+		this, SLOT(onCustomContextMenu(const QPoint &)));
```

Comment:
note for the future: please don't use SIGNAL/SLOT, prefer "modern connection syntax" using pointers to methods

Reply:
> Yes. But it's in line with the other `connect`'s the lines below.

it's not an excuse :)